data/incipias/incipias first contact.txt

```diff
@@ -215,9 +215,9 @@ mission "Incipias: Help The Stranded 1"
 			choice
 				`	"I could help them."`
 					goto initiative
+				`	"How can you make such a cold-hearted decision?"`
```

Comment:
Moving it places the option at the bottom of the choices as I have mentioned in the PR description. You don't need a goto here; the game defaults to going to the next available line. So, clicking on "How can you.." leads directly to the below "It is perhaps...." while the others go to their respective conditions.

I considered reorganizing the labels too, but because "initiative" has a choice following it and decent dialog, it makes more sense to leave them alone.